prompting/miner_availability/miner_availability.py

```diff
@@ -0,0 +1,82 @@
+from pydantic import BaseModel
+from loguru import logger
+from prompting.tasks.base_task import BaseTask
+from prompting.llms.model_zoo import ModelZoo
+from prompting.base.loop_runner import AsyncLoopRunner
+from prompting.base.protocol import AvailabilitySynapse
+from prompting.settings import settings
+from prompting.tasks.date_qa import DateQuestionAnsweringTask
+from prompting.tasks.qa import QuestionAnsweringTask
+from prompting.tasks.summarization import SummarizationTask
+from prompting.tasks.inference import SyntheticInferenceTask, OrganicInferenceTask
+from prompting.utils.uids import get_uids
+import random
+
+task_config: dict[str, bool] = {
+    DateQuestionAnsweringTask.__name__: True,
+    QuestionAnsweringTask.__name__: True,
+    SummarizationTask.__name__: True,
+    SyntheticInferenceTask.__name__: True,
+    OrganicInferenceTask.__name__: True,
+}
```

Comment:
Can we dynamically get the list of tasks from task_registry, to reduce future maintenance?

Reply:
In principle yes, but we will likely want to modify this so that by default most tasks are false. I will switch to dynamically loading it for now, but we will likely have to revert.